test/check-application

```diff
@@ -2578,6 +2578,216 @@ class TestApplication(testlib.MachineCase):
+
+        if auth or self.has_cgroupsV2:
+            b.wait_visible("#run-image-dialog-memory-limit-checkbox:checked")
+            b.wait_visible("#run-image-dialog-memory input[value='256']")
+
+        if auth:
+            b.wait_visible("#run-image-dialog-cpu-priority-checkbox:not(:checked)")
+            b.wait_visible("#run-image-cpu-priority input[value='1024']")
+
+        # Integration tab
+        b.click("#pf-tab-1-create-image-dialog-tab-integration")
+        b.wait_in_text("#pf-tab-section-1-create-image-dialog-tab-integration .publish-port-form", "No ports exposed")
+        b.wait_in_text("#pf-tab-section-1-create-image-dialog-tab-integration .volume-form", "No volumes specified")
+
+        b.wait_visible("#run-image-dialog-env-0-key[value='MYVAR']")
+        b.wait_visible("#run-image-dialog-env-0-value[value='simple_container']")
+        # only one env is set
+        b.wait_not_present("#run-image-dialog-env-1")
+
+        # Healthcheck tab - has default values
+        b.click("#pf-tab-2-create-image-dialog-tab-healthcheck")
+        b.wait_visible("#run-image-dialog-healthcheck-command[value='']")
+        b.wait_visible("#run-image-healthcheck-interval input[value='30']")
+        b.wait_visible("#run-image-healthcheck-timeout input[value='30']")
+        b.wait_visible("#run-image-healthcheck-start-period input[value='0']")
+        b.wait_visible("#run-image-healthcheck-retries input[value='3']")
+
+        b.click("#create-image-create-run-btn")
+        self.waitContainerRow("busybox_copy")
+
+        ports = self.execute(auth, "podman inspect --format '{{.NetworkSettings.Ports}}' busybox_copy")
+        self.assertEqual(ports, 'map[]\n')
+
+        env = self.execute(auth, "podman exec busybox_copy env")
+        self.assertIn('MYVAR=simple_container', env)
+        self.assertIn('container=podman', env)
+        self.assertIn('TERM=xterm', env)
+        self.assertIn('HOME=/root', env)
+        self.assertIn('PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin', env)
+
+        # Clone container and set different values
+        # Switch to Details tab
+        self.performContainerAction("busybox_copy", "Clone")
+
+        # Change memory configuration
+        if auth or self.has_cgroupsV2:
+            b.set_input_text("#run-image-dialog-memory-limit input[type=number]", "1024")
+
+        if auth:
+            # Set cpu shares
+            b.set_checked("#run-image-dialog-cpu-priority-checkbox", True)
+            b.set_input_text("#run-image-dialog-cpu-priority input[type=number]", "2048")
+
+        # Enable tty
+        b.set_checked("#run-image-dialog-tty", True)
+
+        # Change command
+        b.set_input_text('#run-image-dialog-command', "sleep 900")
+
+        if auth:
+            # Add restart policy
+            b.set_val("#run-image-dialog-restart-policy", "on-failure")
+            b.set_input_text('#run-image-dialog-restart-retries input', '2')
```

Comment:
Restart policy set to 2, below we seem to check for 5?

Reply:
![image](https://github.com/cockpit-project/cockpit-podman/assets/67428/c52c5d8d-8714-40df-ac5f-a272ca4de729)

![image](https://github.com/cockpit-project/cockpit-podman/assets/67428/16128388-c5af-49f4-939a-fb57ef87c1df)

So this seems like a bug